perllib/FixMyStreet/App/Controller/Admin/Bodies.pm

```diff
@@ -234,6 +234,14 @@ sub update_contact : Private {
 
     my $category = $self->trim( $c->get_param('category') );
     $errors{category} = _("Please choose a category") unless $category;
+
+    my $body = $c->model('DB::Body')->find({
+        id => $c->stash->{body_id},
+    });
+    my $cobrand = $body->get_cobrand_handler;
+    my $category_validation_error = $cobrand ? $cobrand->call_hook(validate_category => $category) : "";
+    $errors{category} = _($category_validation_error) unless !$category_validation_error;
```

Comment:
A translation call like this won't work, because the thing that looks for strings to translate (which scans the files for wrapped strings) won't find the strings themselves. There's no need to call it here, you'd call it where the string was (if it needed translating, which I guess it doesn't here).

Reply:
I think `if $category_validation_error` slightly clearer than `unless !$category_validation_error` :)